mobile-app/www/js/app.js

```diff
@@ -41,15 +50,38 @@ var app = {
     if (app.turnOffRequests) {
       return;
     }
-
+    adMessage = (app.appScope() == "web") ? ' - via webapp, <http://www.founders-founders.com/doorservices/|get yours here>' : ' - via app, <https://github.com/FoundersFounders/door-services|get yours here>';
     var url =
         'https://slack.com/api/chat.postMessage?token=' + app.token +
         '&channel=' + app.channel +
-        '&text=@door-service: garage&link_names=1' +
+        '&text=@door-service: garage' + adMessage + '&link_names=1' +
         '&as_user=true';
     app.sendAction(url);
   },
 
+  initiateSlackAuth: function() {
+//    window.requestFileSystem(LocalFileSystem.PERSISTENT, 1024, function (fs) {
+//      fs.root.getFile("token", { create: true, exclusive: false }, function (fileEntry) {
+//        app.getToken(fileEntry);
+//      });
+//    }); 
+    var storedToken = window.localStorage.getItem(app.slackAuthTokenKeyName);
+    if (storedToken === null) {
+      app.getOAuthToken();
+  } else {
+    app.token = storedToken;
+    app.showNotification("Found local slack auth token: "+app.token);
```

Comment:
We shouldn't notify the user when things are working as expected. Furthermore, this notification is too large and hides the buttons when the application opens. I suggest removing it.